### PR TITLE
created a method in appoinmentDetailController for fetching a todys a…

### PR DESCRIPTION
**Pull request by kamalahadapadbh**
…ppoinments

### CODE REVIEW

File: data-service/src/main/java/com/nova/dataservice/DTO/AppoinmentDTO.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
+package com.nova.dataservice.DTO;
+
```
Comment (santoshsh2307) [inline data-service/src/main/java/com/nova/dataservice/DTO/AppoinmentDTO.java:2]: please dont use empty space remove this line.

File: data-service/src/main/java/com/nova/dataservice/DTO/AppoinmentDTO.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	public void setShopId(Long shopId) {
+		this.shopId = shopId;
+	}
+	public LocalTime getAppointmentTime() {
+		return appointmentTime;
+	}
+	public void setAppointmentTime(LocalTime appointmentTime) {
+		this.appointmentTime = appointmentTime;
+	}
+
```
Comment (santoshsh2307) [inline data-service/src/main/java/com/nova/dataservice/DTO/AppoinmentDTO.java:73]: remove white space

File: data-service/src/main/java/com/nova/dataservice/controller/AppointmentDetailsController.java [hunk 3]
```diff
@@ -69,5 +72,19 @@ public ResponseEntity<Object> getAllAppontments(@PathVariable("appId") Long id)
...
+			List<AppoinmentDTO> data = appService.getTodaysAppointemtsByLabId(date, labId, fromDate, toDate);
+			if (data.isEmpty()) {
+				return new ResponseEntity<Object>("no data found", HttpStatus.OK);
+			} else {
+				return new ResponseEntity<Object>(data, HttpStatus.OK);
+			}
+		} catch (Exception e) {
+			e.printStackTrace();
+		return new ResponseEntity<Object>("Something went wrong",HttpStatus.OK);
+	}
```
Comment (santoshsh2307) [inline data-service/src/main/java/com/nova/dataservice/controller/AppointmentDetailsController.java:88]: align code properly

File: data-service/src/main/java/com/nova/dataservice/daoimpl/AppointmentDaoImpl.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+				+ "GROUP BY  "
+				+ "    sa.id, ud.username, ud.adhar_photo, ud.phone, ap.appointment_status ";
+		
+		Query query = entityManager.createNativeQuery(sql.toString())
+				.setParameter("toDayDatate", date)
+				.setParameter("labId",  labId ); 
+		
+		query.unwrap(NativeQuery.class)
+		.addScalar("appointmentId",StandardBasicTypes.LONG)
+        .addScalar("patientName", StandardBasicTypes.STRING)
```
Comment (santoshsh2307) [inline data-service/src/main/java/com/nova/dataservice/daoimpl/AppointmentDaoImpl.java:54]: align code properly

File: data-service/src/main/java/com/nova/dataservice/DTO/AppoinmentDTO.java (hunk 1 above)
Comment (kamalahadapadbh) [reply data-service/src/main/java/com/nova/dataservice/DTO/AppoinmentDTO.java:2]: removed

File: data-service/src/main/java/com/nova/dataservice/daoimpl/AppointmentDaoImpl.java (hunk 4 above)
Comment (kamalahadapadbh) [reply data-service/src/main/java/com/nova/dataservice/daoimpl/AppointmentDaoImpl.java]: made changes

**Review by santoshsh2307 — APPROVED** [review]

**Pull request merged**